make.js

```diff
@@ -0,0 +1,15 @@
+function make() {
```

Comment:
since we are using ES6 features, it's better not to use "arguments". 
"function make(...args)" would be better